osgar/drivers/rosmsg.py

```diff
@@ -500,11 +504,6 @@ def slot_raw(self, timestamp, data):
         if frame_id.endswith(b'/base_link/camera_front') or frame_id.endswith(b'camera_color_optical_frame'):
             # used to be self.topic_type == 'sensor_msgs/CompressedImage'
             self.bus.publish('image', parse_jpeg_image(packet))
-        elif frame_id.endswith(b'base_link/front_laser') or frame_id.endswith(b'hokuyo_link'):  #self.topic_type == 'sensor_msgs/LaserScan':
```

Comment:
these parts are still used in SubT and on real robots with ROS/Jetson combination - so you would have to change also related configurations. In that case I would propose to implement first optional renaming output topics